bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -193,8 +183,44 @@ public String formatMagnitude(Magnitude.Band band) {
             }
 
             @Override public Icon getIcon() {
-                if (!isActiveGuideProbe) return errorIcon;
-                else return GuidingIcon.apply(quality.getOrElse(AgsGuideQuality.Unusable$.MODULE$), enabled);
+                return isActiveGuideProbe ?
+                        GuidingIcon.apply(quality.getOrElse(AgsGuideQuality.Unusable$.MODULE$), enabled) :
+                        errorIcon;
+            }
+        }
+
+        static final class BagsTargetRow extends NonBaseTargetRow {
+            final boolean isActiveGuideProbe;
+            final Option<AgsGuideQuality> quality;
+            BagsTargetRow(final boolean isActiveGuideProbe, final Option<AgsGuideQuality> quality, final boolean enabled,
+                          final GuideProbe probe, final SPTarget target, final Option<WorldCoords> baseCoords,
+                          final Option<Long> when) {
+                super(enabled, String.format("%s (BAGS)", probe.getKey()), target, baseCoords, when);
+                this.isActiveGuideProbe = isActiveGuideProbe;
+                this.quality = quality;
+            }
+
+            private static final Icon bagsEnabled  = bagsIcon(true);
+            private static final Icon bagsDisabled = bagsIcon(false);
+            private static final int  bagsGap      = 5;
+
+            private static Icon bagsIcon(boolean enabled) {
+                final BufferedImage img = new BufferedImage(16, 16, BufferedImage.TYPE_INT_ARGB);
```

Comment:
Why do we render the icon instead of using a static image?
Should the icon's size be fixed?